edx_proctoring/views.py

```diff
@@ -386,6 +393,206 @@ def get(self, request):
         return Response(data)
 
 
+class StudentOnboardingStatusByCourseView(ProctoredAPIView):
+    """
+    Endpoint for the StudentOnboardingStatusByCourseView
+
+    Supports:
+        HTTP GET: return information about learners' onboarding status relative to the given course_id
+
+    HTTP GET
+        /edx_proctoring/v1/user_onboarding/status/course_id/{course_id}?page={}&text_search={}&status={}
+
+    **Query Parameters**
+        * page: Optional. The page of the paginated data requested. If this parameter is not provided,
+          it defaults to 1.
+        * text_search: Optional. A search string to perform a case insensitive search for users by either their username
+          or email.
+        * status: Optional. A string representing a status of InstructorDashboardOnboardingAttemptStatus to filter the
```

Comment:
Wait, shouldn't we update the querystring to be `statuses` and update the comments here to accept multiple statuses?